python/lsst/pipe/tasks/finalizeCharacterization.py

```diff
@@ -394,16 +423,18 @@ def run(self, visit, band, isolated_star_cat_dict, isolated_star_source_dict, sr
                 measured_src['visit'][:] = visit
                 measured_src['detector'][:] = detector
 
-                measured_src_tables.append(measured_src.asAstropy().as_array())
+                measured_src_tables.append(measured_src.asAstropy())
 
         if len(measured_src_tables) > 0:
-            measured_src_table = np.concatenate(measured_src_tables)
+            measured_src_table = astropy.table.vstack(measured_src_tables, join_type='exact')
```

Comment:
Not TableVStack? Or these are already in memory so there's no need for it?

Reply:
These are in memory, so there's no need.